polytracker/taint_dag.py

```diff
@@ -385,22 +384,24 @@ def create_node(self, label: int) -> TDTaintForestNode:
             )
 
         elif isinstance(node, TDRangeNode):
-            for n in range(node.first, node.last):
+            curr: int = node.first
+            for n in range(node.first + 1, node.last):
```

Comment:
When storing `last` and `first` in the tdag file, `last` is inclusive but the range-loop is exclusive of last element. Is this accounted for when parsing the range node? Otherwise we probably miss the last element?
Also, I am not all clear about why we start iteration from the second element in the range (`node.first + 1`)?

Reply:
👍